src/main/java/eiteam/esteemedinnovation/armor/ArmorModule.java

```diff
@@ -260,552 +351,550 @@ public static String resource(@Nonnull String base) {
     }
 
     @Override
-    public void recipes(Side side) {
-        if (Config.enableGoggles) {
-            BookRecipeRegistry.addRecipe("monocle1", new ShapedOreRecipe(MONOCLE,
-              " l ",
-              "l l",
-              "btb",
-              'b', INGOT_BRASS,
-              'l', LEATHER_ORE,
-              't', SPYGLASS
-            ));
-            BookRecipeRegistry.addRecipe("monocle2", new ShapedOreRecipe(MONOCLE,
-              " l ",
-              "l l",
-              "btb",
-              'b', PLATE_THIN_BRASS,
-              'l', LEATHER_ORE,
-              't', SPYGLASS
-            ));
-            BookRecipeRegistry.addRecipe("goggles1", new ShapedOreRecipe(GOGGLES,
-              " l ",
-              "l l",
-              "tbg",
-              'b', INGOT_BRASS,
-              'l', LEATHER_ORE,
-              't', SPYGLASS,
-              'g', BLOCK_GLASS_COLORLESS
-            ));
-            BookRecipeRegistry.addRecipe("goggles2", new ShapedOreRecipe(GOGGLES,
-              " l ",
-              "l l",
-              "tbg",
-              'b', PLATE_THIN_BRASS,
-              'l', LEATHER_ORE,
-              't', SPYGLASS,
-              'g', BLOCK_GLASS_COLORLESS
-            ));
+    public void recipes(RegistryEvent.Register<IRecipe> event) {
```

Comment:
The continuation indentations in this method implementation use 8 space tabs instead of 2 space tabs.